packages/react-app/src/lib/constants.js

```diff
@@ -14,10 +18,55 @@ export const POLLING_INTERVAL =
 export const DEFAULT_BRIDGE_DIRECTION =
   process.env.REACT_APP_DEFAULT_BRIDGE_DIRECTION || ETH_XDAI_BRIDGE;
 
+export const NATIVE_CURRENCY_SYMBOLS = ['ETH', 'KETH', 'BNB'];
```

Comment:
We do not need either of these arrays imho. We can use the chainId as key instead of NATIVE_CURRENCY_SYMBOLS, and use foreignChainId instead of NATIVE_CURRENCY_CHAIN_IDS